src/kagglehub/gcs_upload.py

```diff
@@ -66,16 +69,27 @@ def _upload_blob(file_path: str, model_type: str):
         token_exception = "'token' field missing from response"
         raise BackendError(token_exception)
 
+    # Define chunk size (e.g., 1 MB)
+    chunk_size = 1024 * 1024
```

Comment:
Can you move to a constant above and add a comment on why you picked this number?

I would suggest picking the same default chunk size than when you used `data=f` before adding the upload progress.

Reply:
i just 1MB based because it seems to be what people use (based on internet search). we just uploaded the whole file then, there was no default size, am i missing something?

Reply:
1MB seems like a good default.  

Before, you passed a file handle. Under the hood, it is probably uploading in chunks. I looked quickly at the source code but it wasn't trivial to find. Let's use 1MB.

Reply:
Actually, looking at the GCS documentation. They recommend 8MB chunk size: https://cloud.google.com/storage/docs/performing-resumable-uploads#chunked-upload

Reply:
Actually, looking at the doc, they recommend using single chunk upload: https://cloud.google.com/storage/docs/performing-resumable-uploads#upload-data

To show a progress update, you can use a `CallbackIOWrapper`: https://stackoverflow.com/a/73227056